docs/content/configuration/index.md

```diff
@@ -248,6 +248,20 @@ You can enable caching of results at the broker/historical using following confi
 |`druid.cache.maxObjectSize`|Maximum object size in bytes for a Memcached object.|52428800 (50 MB)|
 |`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
 
+#### Redis
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.cache.host`|Redis host|127.0.0.1|
+|`druid.cache.port`|Redis port|6379|
```

Comment:
is it possible to support sharding across multiple servers? For memcached we use consistent hashing on the client side to distribute across multiple hosts.